js/main.js

```diff
@@ -0,0 +1,325 @@
+  btn: 'See Source',
+  side: ['Next Project', 'Previous Project'],
+  icons: ['./assets/icons/Vector.png', './assets/icons/Union.png'],
+  btnForPopup: ['See Live', 'See Source'],
+  nextPrevious: ['./assets/icons/pre.png', './assets/icons/next.png'],
+},
+];
+
+const projectList = document.querySelector('#portfolio ul.grid');
+
+function loadProjects() {
+  for (let i = 0; i < projects.length; i += 1) {
+    const projectItemtemplate = `
+    <li class="project-item ${projects[i].class}">
+      <img
+        class="project-img"
+        src="${projects[i].image}"
+        alt="laptop"
+      />
+      <div class="container container-information hover">
+        <h2 class="heading">${projects[i].title}</h2>
+        <ul class="features">
+          <li class="feature">${projects[i].features[0]}</li>
+          <li class="feature">${projects[i].features[1]}</li>
+          <li class="feature">${projects[i].features[2]}</li>
+        </ul>
+        <button data-id="${projects[i].id}" class="c-projects">
+          See this project &rarr;
+        </button>
+      </div>
+    </li>
+  `;
+
+    projectList.innerHTML += projectItemtemplate;
+  }
+}
+
+window.addEventListener('load', loadProjects());
+
+function createModal(id) {
+  const pCardTemplate = `
+  <div class="overlay hidden">
+      <article class="modal container">
+        <div class="modal__header">
+          <div class="modal__header_title flex space-between">
+            <h1 class="top_title">${projects[id].title}</h1>
+            <button type="button" class="top_btn">
+              &#x2716
+            </button>
+          </div>
+          <ul role="list" class="modal__header_bottom features-container flex">
+            <li class="feature">${projects[id].features[0]}</li>
+            <li class="feature">${projects[id].features[1]}</li>
+            <li class="feature">${projects[id].features[2]}</li>
+          </ul>
+        </div>
+        <div class="modal__body">
+          <div class="modal__body_slide">
+            <div class="slide_main">
+              <span class="slide_prev flex">&lt;</span>
+              <div class="slide_show">
+                <img src="${projects[id].image}" />
+              </div>
+              <span class="slide_next flex">&gt;</span>
+            </div>
+          </div>
+          <div class="modal__body_wrapper">
+            <div class="modal__body_description">
+              <p>${projects[id].description}</p>
+            </div>
+            <div class="modal__body_buttons flex direction">
+              <button class="btn secondary-btn flex" type="button">
+                <span>See Live</span>
+                <img src="./assets/icons/Union.png" alt="see this project" />
+              </button>
+              <button class="btn secondary-btn flex" type="button">
+                <span>See Source</span>
+                <img src="./assets/icons/Vector.png" alt="see this project" />
+              </button>
```

Comment:
- [ ] [OPTIONAL] Great job making changes requested by the previous reviewer 💯, but I think it could be better if you fix the size of the "See live" and "See source" buttons, to stick as much as possible with the Figma template. 🥇
Your buttons:

![image](https://user-images.githubusercontent.com/91301423/203185941-1142779b-532c-410c-9861-90ebce158f0e.png)

Figma buttons:

![image](https://user-images.githubusercontent.com/91301423/203185994-598f605c-b001-4b36-b756-e5ee55cad15e.png)